content_schemas/formats/popular_links.jsonnet

```diff
@@ -0,0 +1,40 @@
+{
+  document_type: "popular_links",
+  base_path: "optional",
+  routes: "optional",
+  rendering_app: "optional",
+  title: "optional",
+  redirects: "optional",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        links_details: {
```

Comment:
This name feels a little repetitive - `details: { links_details: ...`. I appreciate you don't really want just `links`, as that's going to get confused with the whole concept of `links`.

Maybe we could use `link_collection` for this key, whether or not we call the document type itself `link_collection`?